examples/discovery-search-app/.env

```diff
@@ -1,2 +1,6 @@
 SASS_PATH=../../node_modules:src
-REACT_APP_PROJECT_ID=abcdef
\ No newline at end of file
+REACT_APP_PROJECT_ID=REPLACE_ME
+CLUSTER_USERNAME=
+CLUSTER_PASSWORD=
+CLUSTER_PORT=
+CLUSTER_HOST=
```

Comment:
Nitpick: is there a reason all of these values are empty vs "REPLACE_ME"?

Reply:
so that we don't actually try to use `REPLACE_ME` when loading up the environment variables. the fallbacks with `|| 'fallback_value'` will get used when these environment vars aren't provided